.github/workflows/linting.yaml

```diff
@@ -0,0 +1,60 @@
+name: code-style
+
+on:
+  push:
+    branches: "*"
+  pull_request:
+    branches: master
+
+jobs:
+
+  lint-yapf:
+    name: yapf-formatter
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Install yapf
+        run: |
+          python -m pip install yapf
+      - name: yapf Code Formatter
+        run: |
+          yapf --diff --recursive --parallel .
+
+  lint-flake8:
```

Comment:
Based on the above comment I would advocate removing this.

Reply:
👍 Sounds good.